### DISCUSSION TITLE
Task Manager

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#3062] [Feature Requests]
Discuss a built-in task manager for scheduling tasks (code) at times/intervals. This could be used externally (custom code), or internally (backups). This has the added complexity of our Node vs PHP ports... since only Node is a constantly running process, PHP would need to rely on cron jobs, etc.

**Comment by everyx** [comment]
Very exciting features, and If the task done, it would be better if we can receive relevant notifications on the APP side.

**Comment by goetzc** [comment]
It looks like there is already an internal task scheduler:
https://docs.directus.io/extensions/hooks/#schedule

**Comment by rijkvanzanten** [comment]
This exists now in the form of Directus Flows!

**Reply by bbkane** [reply]
https://docs.directus.io/app/flows.html
